mep/books/templates/books/work_detail.html

```diff
@@ -7,33 +7,66 @@
 {% include 'snippets/header.html' with title=work.title style="detail" %}
 {% endblock %}
 
-{% block main-style %}tabbed{% endblock %}
+{% block main-style %}tabbed white{% endblock %}
 
 {% block content %}
-<nav class="tabs" aria-label="sections">
-    <li class="tab active">
-        <span>details</span>
+{# breadcrumbs & tabs #}
+{% include 'snippets/breadcrumbs.html' %}
+<nav class="tabs" aria-label="tabs">
+    <li class="tab" aria-selected="true">
+        <a href="{% url 'books:book-detail' pk=work.pk %}">bibliography</a>
     </li>
     <li class="tab">
-        <a href="#">events</a>
+        <a href="">activities</a>
     </li>
-</nav>
-<section class="details" aria-label="bibliography">
-    {% include 'books/snippets/work_result.html' %}
-
-{% if work.edition_set.exists %}
-<div class="volumes-issues">
-    <h2>Volume/Issue</h2>
-    <ul>
-    {% for edition in work.edition_set.all %}
-    <li>
-        {{ edition.display_html }}
+    <li class="tab">
+        <a href="">cards</a>
     </li>
-    {% endfor %}
-    </ul>
-</div>
+</nav>
 
+{# bibliographic details #}
+<section class="details" aria-label="bibliography">
+    <div class="split">
+        {# creators & publication date #}
+        <dl class="left">
+            {% for creator in work.creator_set.all %}
+            {% ifchanged %}
+            <dt>{{ creator.creator_type.name }}</dt>
+            {% endifchanged %}
+            <dd>{{ creator.person.name }}</dd>
```

Comment:
I can't remember if we discussed/decided - if a creator is a library member, do we link to their member detail page?

Reply:
Seems intuitive, but having links would change the look of the page. Curious what @gissoo thinks.

Reply:
Good point. Let's not include it in this PR/feature. (Could revisit depending on what Gissoo says.)